google_compute_engine/metadata_scripts/script_executor.py

```diff
@@ -25,15 +25,17 @@
 class ScriptExecutor(object):
   """A class for executing user provided metadata scripts."""
 
-  def __init__(self, logger, script_type):
+  def __init__(self, logger, script_type, default_shell='/bin/bash'):
```

Comment:
Please set `default_shell=None` and in the function have `self.default_shell = default_shell or '/bin/bash'`

Reply:
Done